hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/source/TestStreamReadMonitoringFunction.java

```diff
@@ -438,6 +487,20 @@ private void runAsync(
     task.start();
   }
 
+  private Integer intervalBetween2Instants(HoodieTimeline timeline, String instant1, String instant2) {
+    Integer idxInstant1 = getInstantIdxInTimeline(timeline, instant1);
```

Comment:
Can we just filter the timeline with proper range and count the instants instead?

Reply:
not called, just remove it.